http-server/src/test/resources/clientcert-pem/clientcert-pem.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+
+set -eux
```

Comment:
We should move the other script here and name it `clientcert-java.sh`, and also rename the directory to `clientcert-java`